api/prometheus/v1/api.go

```diff
@@ -238,6 +239,8 @@ type API interface {
 	Query(ctx context.Context, query string, ts time.Time) (model.Value, Warnings, error)
 	// QueryRange performs a query for the given range.
 	QueryRange(ctx context.Context, query string, r Range) (model.Value, Warnings, error)
+	// Runtimeinfo returns the various runtime information properties about the Prometheus server.
```

Comment:
```suggestion
	// RuntimeInfo returns the various runtime information properties about the Prometheus server.
```

Reply:
I disagree with this, as the endpoint is /runtimeinfo not /runtime_info, at least that was the logic I used here. :)

Reply:
Interesting, not a blocker, but this looks weird (: